manifests/d/Datalust/Seq/2021.2.6288/Datalust.Seq.yaml

```diff
@@ -0,0 +1,34 @@
+PackageIdentifier: Datalust.Seq
+PackageVersion: 2021.2.6288
```

Comment:
![image](https://user-images.githubusercontent.com/52826317/128823009-99e15a4c-c906-441e-828b-42ecd45370b5.png)

```suggestion
PackageVersion: 6.0.6288.0
```

Reply:
Thanks for the review! The package version is correct at 2021.2.6288; the MSI unfortunately has to use a surrogate version number, because Windows Installer won't allow major version parts > 256 (IIRC).